pkg/logger/logger.go

```diff
@@ -271,7 +272,19 @@ func setLogConf(logConfig string) {
 		_ = ioutil.WriteFile(path, []byte(logConfigContent), os.ModePerm)
 	}
 	fmt.Fprintf(os.Stderr, "load log config %s \n", path)
-	logger, err := seelog.LoggerFromConfigAsFile(path)
+	content, err := os.ReadFile(path)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "init logger error", err)
+		return
+	}
+	dat := string(content)
+	aliyunLogtailLogLevel := strings.ToLower(os.Getenv("ALIYUN_LOGTAIL_LOG_LEVEL"))
+	if aliyunLogtailLogLevel != "" {
+		pattern := `(?mi)(<seelog\s+[^>]*\bminlevel=")[^"]*(("[^>]*>)|("))`
```

Comment:
will this be simpler?
(?mi)(<seelog\s+[^>]*\bminlevel=")[^"]*("[^>]*>)
and we will ReplaceAllString(dat, `${1}`+aliyunLogtailLogLevel+`${2}`)

Reply:
Seems make no effect， I use ``(?mi)(minlevel=")([^"]*)(")``  instead